code/app.js

```diff
@@ -0,0 +1,8 @@
+function menuToggle() {
+  var menu = document.getElementById("main-menu-cnt");
```

Comment:
Let's try to remember about avoiding `var` keyword - there is really small amount of use cases when we should use it :) By default, we should aim for `const`